ross/disk_element.py

```diff
@@ -232,16 +235,16 @@ def patch(self, position, ax, bk_ax):
             color=bokeh_colors[9]
         )
         bk_ax.circle(
-            x=zpos,
-            y=ypos + D,
-            radius=0.02,
+            x=bk_disk_points_u[0][0],
```

Comment:
The indices here (`[0][0]`) make the code hard to read. Maybe this could be better with a dictionary? See if it is possible to improve it.